spec/datetimeformat.html

```diff
@@ -91,9 +91,13 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
           1. Set _timeZone_ to DefaultTimeZone().
         1. Else,
           1. Set _timeZone_ to ? ToString(_timeZone_).
-          1. If the result of IsValidTimeZoneName(_timeZone_) is *false*, then
-            1. Throw a *RangeError* exception.
+        1. If IsTimeZoneOffsetString(_timeZone_) is *true*, then
+          1. Let _offsetNanoseconds_ be ParseTimeZoneOffsetString(_timeZone_).
```

Comment:
There's a new CanonicalizeTimeZoneOffsetString AO in Temporal. Should this be brought into 402 to replace this line and the one below it?

Reply:
I do not change the definition of HourSubcomponents in ECMA262 instead
Currently, it is 
```
HourSubcomponents[Extended] :::
  TimeSeparator[?Extended] MinuteSecond
  TimeSeparator[?Extended] MinuteSecond TimeSeparator[?Extended] MinuteSecond TemporalDecimalFractionopt
```
why not change it to just
```
HourSubcomponents[Extended] :::
  TimeSeparator[?Extended] MinuteSecond
```

?

Reply:
The only place reference to  it is  UTCOffset and the only places refence to UTCOffset are 
21.4.1.33.1 IsTimeZoneOffsetString ( offsetString )
and
21.4.1.33.2 ParseTimeZoneOffsetString ( offsetString )

which are exactly the two places we need to change and both places are only used by place dealing with system timezone.

Reply:
@FrankYFTang An ECMA-262 grammar cannot be changed from an ECMA-402 pull request. But all of this _will_ be simplified by/in response to Temporal, as demonstrated in https://github.com/tc39/ecma402/pull/788#discussion_r1320356161 .

Reply:
One thing I do not understand is why do we have this PR for ECMA402 instead of just let Temporal to change both ECMA262 and 402. What is the urgent need for that to happen before Temporal reach stage 4?

Reply:
AFAIK, the main reason is that offset time zones are currently supported in (pre-Temporal) ECMA-262. My assumption is that all ECMA-262 features should be supported when ECMA-402 is also implemented.

Also, do we know yet whether ICU and/or CLDR changes will be needed to enable formatting of offset time zones?  If no, that's great news. If yes, then where do changes need to be made?